swri_image_util/package.xml

```diff
@@ -20,6 +20,8 @@
   <depend>geometry_msgs</depend>
   <depend>image_geometry</depend>
   <depend>image_transport</depend>
+  <depend>libqt4-opengl-dev</depend>
```

Comment:
They don't conflict at all, and in fact work fine when installed side-by-side.  It's mostly just annoying that rosdep is guaranteed to install a version of Qt that you may not be using at all.